tests/testdata/qgis_server/api/test_wfs3_collections_project.json

```diff
@@ -45,16 +49,21 @@ Content-Type: application/json
       ],
       "description": "A Layer1 with an abstract",
       "extent": {
-        "crs": "http://www.opengis.net/def/crs/OGC/1.3/CRS84",
-        "spatial": [
-          [
-            8.203459307036344,
-            44.90139483904469,
-            8.203546993993488,
-            44.901482526001836
-          ]
-        ]
+        "spatial": {
+          "bbox": [
```

Comment:
there's an extra level of array here. Should be `"bbox": [ [ 8.203459307036344, 44.90139483904469, 8.203546993993488,44.901482526001836 ]]`

Reply:
Fixed, thank you for the review!